packages/core/src/tests/fixtures/cjs/src/index.js

```diff
@@ -0,0 +1,24 @@
+const fs = require("fs/promises");
+const { Document, VectorStoreIndex } = require("llamaindex");
```

Comment:
When we do the require here, which llamaindex version is it importing from? Is there a way to make these tests run against the current branch/build in CI rather than a version that's newly deployed?

Reply:
Yes! This is actually using the locally built version directly from the `dist` folder to ensure its both up to date and as close to how consumers will be using it via npm. The magic is done [in the tests by npm linking](https://github.com/run-llama/LlamaIndexTS/pull/147/files#diff-979ca8f0a35ef7412680b8265e6eb93ce02a5c2bc4df167547ba5217d1d14d06R8) the llamaindex package which ensures its using the local copy of it.

Reply:
Awesome!